packages/eslint-plugin-jest/src/index.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
         it: false,
         jasmine: false,
         jest: false,
+        pending: false,
```

Comment:
I honestly didn't realize we exposed this global…

Reply:
I have to admit I wasn't even aware of it myself until I checked out the docs for Jasmine